docs/cql/functions.rst

```diff
@@ -266,6 +266,17 @@ Behavior for null input values must be defined for each function:
 * ``RETURNS NULL ON NULL INPUT`` declares that the function will always return null (without being executed) if any of the input arguments is null.
 * ``CALLED ON NULL INPUT`` declares that the function will always be executed.
 
+The ``AS`` clause defines the function body. The body is a string literal containing the function code:
+
+* For Lua functions, the body is a Lua script.
+
+* For Wasm functions, the body is a WebAssembly module in WebAssembly text format, which exports a function with the same name as specified in the ``CREATE FUNCTION`` statement.
+    More details on generating the Wasm modules can be found `here </kb/wasm>`_.
```

Comment:
```suggestion
* For Wasm functions, the body is a WebAssembly module in WebAssembly text format, which exports a function with the same name as specified in the ``CREATE FUNCTION`` statement. More details on generating the Wasm modules can be found `here </kb/wasm>`_.
```

Reply:
The text must be on one line; otherwise, it renders with errors.

Reply:
Ok